### ISSUE TITLE
User dictionry not working?

### ISSUE THREAD

**Issue by glukash** [gordio/ToggleWords#13]
User dictionary createg but it doesn't toggle.

**Comment by gordio** [comment]
@glukash please, can you share you config file?

**Comment by MrStaind** [comment]
+1

**Comment by gordio** [comment]
@MrStaind please, can you show config files?

**Comment by MrStaind** [comment]
which config? 
when i create /package/ToggleWords/ToggleWords.sublime.settings
whole plugin stops working

**Comment by MrStaind** [comment]
ToggleWords.sublime-settings
```
{
    // User defined words
    "toggle_word_dict": [
        ["left", "right"],
        ["up", "down"],
        ["top", "bottom"],
        ["width", "height"],
        ["red","orange","yellow","green","blue","purple"],
        ["true", "false"],
        ["yes", "no"],
        ["on", "off"],
        ["0", "1"],
        ["new", "word"]
    ]
}

```

**Comment by MrStaind** [comment]
i just added arrays to source code for now.

**Comment by gordio** [comment]
@MrStaind thank you, I check problem after work (dead line coming, don't have time :) )

**Comment by webcaetano** [comment]
Same problem.

**Comment by gordio** [comment]
I change plugin name. As I think this is problem. But if you now have problem, please reinstall plugin.

**Comment by Alseyn** [comment]
I just installed this from package control and had the same issue. I'm using Windows running build 3114 of Sublime. Looking through the code I saw it looks for `toggle_words_dict` but the instructions say `toggle_word_dict`, the default keyboard shortcut is `ctrl-alt-x` rather than `win-alt-x` too. I didn't find the message about the name change telling people to rename the setting until I'd already done it. It might be a good idea to update the instructions! :)

Also, Trying to open the default settings or keymap files from the package settings menu gives me an error. It tries to open them from `%appdata%\Sublime Text 3\Packages\ToggleWords\` when the package is located at `%appdata%\Sublime Text 3\Installed Packages\Toggle Words.sublime-package` I added the defaults I wanted to my user settings and got things working.

Thanks for this. I think I was beginning to get true-or-false poisoning.
